src/containers/Inventory.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { connect } from 'react-redux'
+import { changeBlockName, toggleProperties } from '../actions'
+import Inventory from './../components/Inventory.js';
```

Comment:
That's just trying to find the correct path to the file. I could probably make it more relative and less absolute.